pinot-spi/src/main/java/org/apache/pinot/spi/trace/RequestContext.java

```diff
@@ -19,6 +19,7 @@
 package org.apache.pinot.spi.trace;
 
 public interface RequestContext {
+  RequestContext getRequestContext();
```

Comment:
What does this API mean?

Reply:
+1. this seems problematic

Reply:
Umm if we don't expose a getter to RequestContext how would we use it on the listener side?